src/DimensionReduction/pca.jl

```diff
@@ -25,5 +25,6 @@ function fit(ds::DataSet, pca::PCA)
     catch
         error("No local descriptors found in DataSet")
     end
```

Comment:
This function addresses dimension reduction over the global descriptors, but do not addresses force descriptors. Perhaps in a future PR this feature can be considered. I developed a variant of this code that tackle both, energies and forces [here](https://github.com/cesmix-mit/PotentialLearning.jl/blob/4bac004a64da1772550c3edba7a6de6d1d90f283/examples/ACE/utils/PotentialLearningExtensions.jl#L86).